dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java

```diff
@@ -142,40 +157,85 @@ public static void deleteMultiTreeByParent1(Identifier parent, Long languageId)
                     .append("AND c.lang = ").append(languageId).append(");");
 
             db.executeStatement(query.toString());
+            
+            updateVersionTs(parent.getId(), languageId);
+            refreshPageInCache(parent.getId(), languageId);
+            
         } catch (SQLException e) {
             throw new DotDataException("Error deleting tree and multi-tree dependencies.", e);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Error deleting tree and multi-tree dependencies." 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)